src/cli/backup/onedrive.go

```diff
@@ -394,13 +395,14 @@ func runDetailsOneDriveCmd(
 		return nil, errs.Fail(err)
 	}
 
-	d, _, err := r.BackupDetails(ctx, backupID)
-	if err != nil {
-		if errors.Is(err, kopia.ErrNotFound) {
+	d, _, errs := r.BackupDetails(ctx, backupID)
+	// TODO: log/track recoverable errors
+	if errs.Err() != nil {
+		if errors.Is(errs.Err(), kopia.ErrNotFound) {
 			return nil, errs.Fail(errors.Errorf("no backup exists with the id %s", backupID))
 		}
 
-		return nil, errs.Fail(errors.Wrap(err, "Failed to get backup details in the repository"))
+		return nil, errs.Fail(errors.Wrap(errs.Err(), "Failed to get backup details in the repository"))
```

Comment:
Should the original error be included in L401?

Reply:
It is the original error